src/test/java/edu/cornell/kfs/sys/util/MockDocumentUtils.java

```diff
@@ -0,0 +1,97 @@
+package edu.cornell.kfs.sys.util;
+
+import java.util.ArrayList;
+import java.util.function.Consumer;
+
+import org.easymock.EasyMock;
+import org.kuali.kfs.krad.bo.AdHocRoutePerson;
+import org.kuali.kfs.krad.bo.AdHocRouteRecipient;
+import org.kuali.kfs.krad.bo.Note;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.sys.businessobject.FinancialSystemDocumentHeader;
+import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
+import org.kuali.kfs.sys.businessobject.TargetAccountingLine;
+import org.kuali.kfs.sys.document.AccountingDocument;
+import org.kuali.rice.kim.impl.identity.PersonImpl;
+
+import edu.cornell.kfs.fp.batch.xml.fixture.AccountingDocumentClassMappingUtils;
+
+public class MockDocumentUtils {

```

Comment:
this is a great utility class, Perhaps this should be discussed at a future mops team meeting so everyone can know about it.

Reply:
Ok, I'll look into adding a MOPS or bookclub agenda item to demonstrate this further.